src/engine.ts

```diff
@@ -442,30 +440,30 @@ export default class Engine {
     const offers = cmd.data.offers;
 
     const pl = this.player(this.playerToMove);
-    // const jsonData = pl.data.toJSON();
+    const power = Reward.parse(offers[0].offer)[0].count;
 
     // A passed player in last round should always decline a leech if there's a VP
     // cost associated with it.
-    // If this not truth, please add an example (or link to) in the comments
+    // If this not true, please add an example (or link to) in the comments
     if (this.isLastRound && this.passedPlayers.includes(pl.player)) {
       if (!offers.some(offer => offer.cost === '~')) {
-        this.move(`${pl.faction} ${Command.Decline} ${offers[0].offer}`, false);
+        this.move(`${pl.faction} ${Command.Decline} ${power}`, false);
         return true;
       }
     }
 
     // Only leech when only one option and cost is nothing
-    if (offers.length > 1 || Reward.parse(offers[0].offer)[0].count > (pl.data.autoChargePower || 1) ) {
+    if (offers.length > 1 || power > (pl.data.autoChargePower || 1) ) {
```

Comment:
```suggestion
    if (offers.length > 1 || power > (pl.data.autoChargePower ?? 1) ) {
```

Use the nullish coalescing operator